### PR TITLE
Tutorial and usage update

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/180
Fix https://github.com/PaddlePaddle/cloud/issues/2

### CODE REVIEW

File: doc/tutorial_cn.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# 提交第一个训练任务
+
+---
+
+## 下载并配置paddlecloud
+
+`paddlecloud`是提交PaddlePaddleCloud分布式训练任务的命令行工具。
+
+步骤1: 访问链接 https://github.com/PaddlePaddle/cloud/releases 下载最新的`paddlecloud`二进制客户端，并把`paddlecloud`拷贝到环境变量$PATH中的路径下，比如：`/usr/local/bin`
+
```
Comment (dzhwinter) [inline doc/tutorial_cn.md]: do we need to put a  `chmod +x bin` here explicity to teach fresh new user?

Comment (typhoonzero) [inline doc/tutorial_cn.md]: Good point! Done.

File: doc/tutorial_cn.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+wget https://github.com/PaddlePaddle/cloud/blob/develop/demo/fit_a_line/train.py
+cd ..
+paddlecloud submit -jobname fit_a_line -cpu 1 -gpu 1 -parallelism 1 -entry "python train.py" fit_a_line/
+```
+
+可以看到在提交任务的时候，我们指定了任务的名称`-jobname fit_a_line`、使用的CPU资源`-cpu 1`、
+使用的GPU资源`-gpu 1`、并行度`-parallelism 1`(训练节点个数)，启动命令`-entry "python train.py"`
+和任务程序目录`fit_a_line/`。
+
+***说明：*** 如果希望查看完整的任务提交参数说明，可以执行`paddlecloud submit -h`。
```
Comment (Yancey1989) [inline doc/tutorial_cn.md]: 说明1， 说明2

File: doc/tutorial_cn.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+```
+paddlecloud file ls /pfs/dlnel/home/wuyi05@baidu.com/jobs/fit_a_line/
+train.py
+image
+output
+paddlecloud file ls /pfs/dlnel/home/wuyi05@baidu.com/jobs/fit_a_line/output/
+pass-0001.tar
+...
+paddlecloud file get /pfs/dlnel/home/wuyi05@baidu.com/jobs/fit_a_line/output/pass-0001.tar
```
Comment (Yancey1989) [inline doc/tutorial_cn.md]: Missing local filename ?

File: doc/usage_cn.md [hunk 4]
```diff
@@ -173,43 +179,6 @@ paddlecloud submit -jobname my-paddlecloud-job \
 - `-passes`：执行训练的pass个数
 - `package`：HDFS 训练任务package的路径
 
-### 使用自定义的Runtime Docker Image
```
Comment (Yancey1989) [inline doc/usage_cn.md]: 我们不需要自定义runtime Docker Image么？

File: doc/usage_cn.md [hunk 5]
```diff
@@ -132,30 +155,13 @@ scp -r my_training_package/ user@tunnel-server:/mnt/hdfs_mulan/idl/idl-dl/mypack
...
-  -psmemory 1Gi \
-  -passes 1 \
-  -topology trainer_config.py /pfs/[datacenter_name]/home/[username]/ctr_demo_package
+paddlecloud submit -jobname my-paddlecloud-job -cpu 1 -gpu 0 -memory 1Gi -parallelism 10 -pscpu 1 -pservers 3 -psmemory 1Gi -passes 1 -topology trainer_config.py /pfs/[datacenter_name]/home/[username]/ctr_demo_package
 ```
 
 - 提交基于V2 API的训练任务
 
 ```bash
-paddlecloud submit -jobname my-paddlecloud-job \
```
Comment (Yancey1989) [inline doc/usage_cn.md]: Maybe the previous style is convenient to COPY && PASTE.

File: doc/usage_cn.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ Use "paddlecloud flags" for a list of top-level flags
 
 不同的PaddlePaddleCloud集群环境会提供不同的分布式存储服务。目前PaddlePaddleCloud支持HDFS和CephFS。
```
Comment (Yancey1989) [inline doc/usage_cn.md]: What's the relation of *usage_cn.md* and *tutorial.md*? *usage_cn.md* is for internal cluster and another one is for DLNEL?

File: doc/tutorial_cn.md (hunk 2 above)
Comment (typhoonzero) [inline doc/tutorial_cn.md]: Done

File: doc/tutorial_cn.md (hunk 3 above)
Comment (typhoonzero) [inline doc/tutorial_cn.md]: Done

File: doc/usage_cn.md (hunk 6 above)
Comment (typhoonzero) [inline doc/usage_cn.md]: tutorial是类似quick_start的东西，快速使用和上手的教程。只涉及最少的使用流程。usage中将包含更详细的使用说明

**Pull request merged**